### ISSUE TITLE
stacked does not provide sum 

### ISSUE THREAD

**Issue by shilpapadgaonkar** [openshift/origin-metrics#35]
Hello,
As suggested in docs-- stacked: statistics are first computed for every matching metric and then those statistics are stacked (added), I should get sum of memory usage if I am using only a descriptor memory usage for a tenant.
But as you can see in my example below, the only change i see is in the number of samples

[shilpa@vmapgmstapp2n1 ~]$ curl -H "Authorization: Bearer NLtkT4-4og6BZEauh4cFKX1dxyX77vG-TrLc1GwdRI4" -H "Hawkular-tenant: scalingsample" -H "Accept: application/json" -X GET https://172.30.231.219/hawkular/metrics/gauges/data?'tags=descriptor_name:memory/usage&bucketDuration=8h&stacked=true' --insecure | python -m json.tool

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   192    0   192    0     0    741      0 --:--:-- --:--:-- --:--:--   741

[
    {
        "avg": 403436606.6938775,
        "empty": false,
        "end": 1447322513238,
        "max": 457138176.0,
        "median": 398397415.5333509,
        "min": 361869312.0,
        "percentile95th": 448614400.0,
        "samples": 1,
        "start": 1447293713238
    }
]
[shilpa@vmapgmstapp2n1 ~]$ curl -H "Authorization: Bearer NLtkT4-4og6BZEauh4cFKX1dxyX77vG-TrLc1GwdRI4" -H "Hawkular-tenant: scalingsample" -H "Accept: application/json" -X GET https://172.30.231.219/hawkular/metrics/gauges/data?'tags=descriptor_name:memory/usage&bucketDuration=8h&stacked=false' --insecure | python -m json.tool

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   190    0   190    0     0   1042      0 --:--:-- --:--:-- --:--:--  1043

[
    {
        "avg": 404510638.08,
        "empty": false,
        "end": 1447322544621,
        "max": 457138176.0,
        "median": 394453166.6205565,
        "min": 361869312.0,
        "percentile95th": 452808704.0,
        "samples": 50,
        "start": 1447293744621
    }
]

**Comment by mwringe** [comment]
I see this was closed. Did you figure out what the issue was?

**Comment by shilpapadgaonkar** [comment]
No, I closed this since I thought that the purpose of stacked was not to provide the sum.
Earlier I had assumed that if I take a stacked=true metrics of 3 pods running under tenant A and if max memory usage of pod1 = 2gb, pod2 = 3gb and pod3=5gb, the max value I get in return would be (2+3+5)=10 gb. 

But then i realized  that probably the only difference between uniform and stacked is that
uniform would take metrics from all these pods as if from a single source and perform statistics on them, while stacked would take them as metrics from diff sources, perform stats and then put them together. But here, since the documentation mentioned the word "adds" them, I assumed it would sum them up. 

Hence I closed the ticket.
